.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['**']
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        ruby: [2.6, 2.7, '3.0', jruby, truffleruby]
```

[note] I'd like to add also Ruby 3.1 like this,

```suggestion
        ruby: [2.6, 2.7, '3.0', 3.1, jruby, truffleruby]
```

but it's impossible due to the old RuboCop version:

https://github.com/sickill/rainbow/blob/ea58198db11f9eee8c2dd75cd31470e7aead86bf/Gemfile#L19

It's possible to add 3.1 to the CI matrix if we could drop the support of Ruby 2.3.

---

I think a drop of 2.3 support is good in time, now.

---

I've updated RuboCop and `.rubocop.yml` disabling some cops to avoid RuboCop errors and warnings.